CMakeLists.txt

```diff
@@ -12,25 +12,8 @@ find_package(Boost REQUIRED system filesystem)
 find_package(ASSIMP QUIET)
 if (NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
-  pkg_check_modules(ASSIMP assimp)
-endif()
-if (ASSIMP_FOUND)
```

Comment:
Hm, I made these proposals under the impression
that we do not support indigo on ubuntu 13.10 anymore
as it is officially unsupported for two years now and
osrf stopped building debs for it.

As far as I'm concerned, I believe we will be stuck with
indigo for quite some more time. Many low-level packages
were still missing in kinetic last time I checked
and I don't know of many groups using kinetic with
their setups yet. This is why I tend to apply all
ABI stable patches to indigo before moving them.

Any comments/opinions on either point?